src/pdf.js

```diff
@@ -28,6 +28,7 @@ if (typeof PDFJS === 'undefined') {
 //#if BUNDLE_BUILD
 //#expand PDFJS.build = '__BUNDLE_BUILD__';
 //#endif
+//#expand PDFJS.documentID = 0;
```

Comment:
Define documentID as a variable in the api.js (it will be hidden by closure), not need to expose that in the PDFJS namespace.